tests/test_routes.py

```diff
@@ -24,11 +24,11 @@
 from unittest import TestCase
 from uuid import uuid4
 from datetime import datetime, timezone
+import uuid
 from wsgi import app
 from service.common import status
 from service.models import db, Promotion
 from .factories import PromotionFactory
-import uuid
```

Comment:
I noticed that here is 2 import of uuid. I will make another pull request to solve it with the internal error 500 testcase.

Reply:
sure, these 2 import seems to not causing conflicts.